src/adapt/csscasc.js

```diff
@@ -1337,6 +1336,98 @@ adapt.csscasc.CheckConditionAction.prototype.getPriority = function() {
     return 5;
 };
 
+/**
+ * @constructor
+ * @extends {adapt.csscasc.ChainedAction}
```

Comment:
Probably it does not need to be a `ChainedAction`. A `CascadeAction` will do the job.